lightspeed/modules/con_wca-key-model-id.adoc

```diff
@@ -3,19 +3,31 @@
 [id="wca-key-model-id_{context}"]
 = Configuration requirements
 
-To use the {LightspeedShortName} cloud service, your organization must have the following subscriptions:
+== Licensing requirements
 
-* A trial or paid subscription to {PlatformName} 
+* *{LightspeedShortName} cloud service*
++
+To use the {LightspeedShortName} cloud service, you must meet *one* of the following requirements:
 
-* A trial or paid subscription to {ibmwatsonxcodeassistant}
+** Your organization has a trial or paid subscription to both the {PlatformName} and {ibmwatsonxcodeassistant}.
+** Your organization has a trial or paid subscription to the {PlatformName}, and you have an {LightspeedShortName} trial account.
```

Comment:
```suggestion
** Your organization has a trial or paid subscription to the {PlatformName}, and you have a {LightspeedShortName} trial account.
```
This is such a minor nit, but it looks like the attribute will display as Red Hat Ansible Lightspeed, in which case the article should be "a" instead of "an". It this is too nit-picky, you can ignore. Otherwise, I suggest doing a search and replace.

Reply:
Great catch, @jself-sudoku  and thank you. I updated the docs globally.